<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LedWin</class>
 <widget class="QMainWindow" name="LedWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget{
	border:2px solid#d965ff;
	border-image: url(:/ç»¿.jpg);
}

QPushButton:focus{
	background-color: rgb(252, 233, 79);
}

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="1" column="1">
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1,2">
      <item>
       <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,2" columnminimumwidth="0,0">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLabel" name="label">
                <property name="styleSheet">
                 <string notr="true">border:2px solid#d965ff;</string>
                </property>
                <property name="text">
                 <string>name</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_2">
                <property name="styleSheet">
                 <string notr="true">border:2px solid#d965ff;</string>
                </property>
                <property name="text">
                 <string>ISDN</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_3">
                <property name="styleSheet">
                 <string notr="true">border:2px solid#d965ff;</string>
                </property>
                <property name="text">
                 <string>status</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QLineEdit" name="nameEdit">
                <property name="styleSheet">
                 <string notr="true">border:2px solid#d965ff;
</string>
                </property>
                <property name="text">
                 <string>LED1</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="ISDNEdit">
                <property name="styleSheet">
                 <string notr="true">border:2px solid#d965ff;</string>
                </property>
                <property name="text">
                 <string>LED-A001</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="statusEdit">
                <property name="styleSheet">
                 <string notr="true">border:2px solid#d965ff;</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="AnimationButton" name="addBtn" native="true">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>120</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>120</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="AnimationButton" name="searchBtn" native="true">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>120</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>120</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="AnimationButton" name="deleteBtn" native="true">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>120</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>120</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 0, 0);
</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="AnimationButton" name="backBtn" native="true">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>120</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>120</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="ledWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>150</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>700</width>
          <height>300</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border:5px solid#3a57ff;
color: rgb(0, 0, 255);

</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="2">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="1">
     <spacer name="verticalSpacer_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <action name="actionsearch">
   <property name="text">
    <string>searchDevice</string>
   </property>
  </action>
  <action name="actionadd">
   <property name="text">
    <string>addDevice</string>
   </property>
  </action>
  <action name="actiondelete">
   <property name="text">
    <string>deleteDevice</string>
   </property>
  </action>
  <action name="actionexit">
   <property name="text">
    <string>exit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AnimationButton</class>
   <extends>QWidget</extends>
   <header>animationbutton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
